README.md

```diff
@@ -37,6 +37,16 @@ All of this work is made possible by [CUGOS](http://cugos.org), an open-source g
 
 **NOTE:** Things are moving very quickly with DNC right now. We are still continuing to build out and refactor the architecture of the application. If you want to read more about our decision-making process take a look at some of our [meeting notes in the wiki](https://github.com/cugos/drop-n-chop/wiki/Meeting-Notes---04-11-2015).
 
+Run a local server with a development (not minified) version of the code:
+
```

Comment:
Actually, maybe these additions to the `README.md` should be rolled into the text on line `36` of the file